Readme benefits and proxy suggestion (#29)
chaincase-app/nolooking: README.md

# L[ightningPayJ]oin: Automated External Lightning Channel Funding

> Loin leverages Pay-to-Endpoint (payjoin) to negotiate a channel open for your lightning node, initiated by any BIP78 supporting wallet. Previously, custom PSBT channel funding could only be done by [signing PSBTs manually](https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md). A node running Loin can open a new lightning channel for every received payment, provided the payer supports bip78. E.g. BTCPayServer or Blue wallet, following the normal QR scan transfer flow. Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github) to hear all the benefits!

Reviewer: ```suggestion
Loin leverages Pay-to-Endpoint ([BIP78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) PayJoin) to negotiate channel establishment on your lightning node from any [supporting wallet](https://en.bitcoin.it/wiki/PayJoin_adoption). Previously, custom PSBT channel funding could only be done by [signing PSBTs manually](https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md). A node running Nolooking can open many lightning channels for every inbound transaction, provided the payer supports BIP78. E.g. BTCPayServer or Wasabi by following the normal QR scan transfer flow. Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github) to hear how it can save you time and money.
```

- Channel Establishment is the lingo the use in the [Bolt 02 spec](https://github.com/lightning/bolts/blob/master/02-peer-protocol.md#channel-establishment)
- stay consistent with bippersz
- BlueWallet not tested yet, nor JM. Is it? Wasabi is. Happy to add as soon as it's tested

Author: Ohh this is a tricky one. My reaction just then was that I had never heard of channel establishment. Do we want to go legit or colloquial? I am leaning towards "Channel open" or "channel funding" because that is familiar phrasing to users

Reviewer: Familiar is OK with me. would just be good to remove Blue because that does not work as teested

## ⚠️ WARNING: ULTRA-EXPERIMENTAL SOFTWARE

❗️ **This project is pre-alpha quality and lacks basic checks. It is NOT fully BIP78-compliant** ❗️

Any unexpecected node version will likely cause permanent loss of funds.

For the love of god, do NOT use it on mainnet just yet!
If you don't want to wait long, help with reviews and PRs

> ### Expected fee savings

Reviewer: ```suggestion
## Expected fee savings
```
I don't think this is subset of ##warning

In the traditional path for opening a lightning channel, you first must fund the LND wallet and then make a second transaction to fund the opening of the channel, a total of two transactions.

> Loin however does this all in a single transaction, saving 106 vB (68 vB input + 22 vB script pubkey + 8 vB output amount + 4 vB version + 4vB timelock).

Reviewer: ```suggestion
Nolooking however does this all in a single transaction, saving **106 vB** (68 vB input + 22 vB script pubkey + 8 vB output amount + 4 vB version + 4vB timelock).
```

> If you wanted to fund a number of channels, say 10 channels, this would traditionally take 11 transactions. Loin can do this in a single transaction - saving 1060 vB.

Reviewer: ```suggestion
If you wanted to fund a number of channels, say 10 channels, this would traditionally take 11 transactions. Nolooking can do this in a single transaction saving 1060 vB.
```

> You can also schedule channel opens, such that when someone goes to pay you lightning channel funding transactions piggyback along with their payment. Again not only saving fees and also saving you a shedload of time since you don't have to wait for confirmations inbetween each successive channel open.

Reviewer: ```suggestion
You can also schedule channel opens, such that when someone goes to pay you lightning channel funding transactions piggyback along with their payment. Again not only saving fees and also saving you a shedload of time since you don't have to wait for confirmations in between each successive channel open.
```

# L[ightningPayJ]oin: Automated External Lightning Channel Funding

Nolooking leverages Pay-to-Endpoint ([BIP78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) PayJoin) to negotiate a channel open for your lightning node from any BIP78 [supporting wallet](https://en.bitcoin.it/wiki/PayJoin_adoption). Previously, custom PSBT channel funding could only be done by [signing PSBTs manually](https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md). A node running nolooking can open many new lightning channel with every inbound transaction, provided the payer supports BIP78. E.g. by using Sparrow wallet, BTCPayServer, or Wasabi; following the normal QR scan and payjoin payment flow. Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github) to hear how it can save your privacy, time, and money!

## ⚠️ WARNING: ULTRA-EXPERIMENTAL SOFTWARE

❗️ **This project is pre-alpha quality and lacks basic checks. It is NOT fully BIP78-compliant** ❗️

> Any unexpecected node version will likely cause permanent loss of funds. 

Reviewer: extra space
